tools/proteore_topgo/topGO.xml

```diff
@@ -0,0 +1,268 @@
+<tool id="topGO" name="Enrichment analysis" version="2021.04.12">
```

Comment:
please use for the tool ID only small letters, see our recommendations here: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-ids

Reply:
Maybe as name "GO enrichment analysising" and as decription `with topGO`